src/FilterList.js

```diff
@@ -173,41 +170,50 @@ let FilterList = _.flow(
               style={bdJoin(child)}
             />
           ) : (
-            <div key={child.path} className="filter-list-item">
-              <Label tree={tree} node={child} fields={fields}>
-                {mapNodeToLabel(child, fields)}
-              </Label>
-              {!child.paused && (
-                <div className="filter-list-item-contents">
-                  <Dynamic
-                    {...{
-                      component: UnmappedNodeComponent,
-                      tree,
-                      node: child,
-                      path: _.toArray(child.path),
-                      ...mapNodeToProps(child, fields),
-                    }}
-                  />
-                  {!child.updating &&
-                    tree.disableAutoUpdate &&
-                    // find if any nodes in the tree are marked for update (i.e. usually nodes are marked for update because they react to "others" reactor)
-                    _.some(
-                      treeNode => treeNode !== node && treeNode.markedForUpdate,
-                      F.treeToArray(_.get('children'))(tree.tree)
-                    ) && (
-                      <div
-                        className="apply-filter-button"
-                        onClick={e => {
-                          e.stopPropagation()
-                          tree.triggerUpdate()
-                        }}
-                      >
-                        <Button primary>Apply Filter</Button>
-                      </div>
-                    )}
-                </div>
-              )}
-            </div>
+            <Expandable
+              key={child.path}
+              className="filter-list-item"
+              isOpen={!child.paused}
+              Label={
+                <Label tree={tree} node={child} fields={fields}>
+                  {mapNodeToLabel(child, fields)}
+                </Label>
+              }
+              onClick={() =>
+                tree &&
+                node &&
```

Comment:
this isn't the right check - the prior implementation checked tree and node because it was toggling node.paused at node.path. This now uses child, so it should check for child instead, right?

Reply:
Yeah that's right

Reply:
I am skipping the child check tho, it's always present in given context